0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/GenerateCommand.scala

```diff
@@ -257,24 +331,31 @@ case class GenerateCommand(
    * Output end result
    */
   def outputResult(result: DdlOutput): Unit = {
-    result.ddls
-      .map(_.setBasePath("sql"))
-      .map(_.setBasePath(output.getAbsolutePath))
-      .map(_.write(force)).foreach(printMessage)
-
-    result.jsonPaths
-      .map(_.setBasePath("jsonpaths"))
-      .map(_.setBasePath(output.getAbsolutePath))
-      .map(_.write(force)).foreach(printMessage)
-
-    result.migrations
-      .map(_.setBasePath("sql"))
-      .map(_.setBasePath(output.getAbsolutePath))
-      .map(_.write(force)).foreach(printMessage)
-
-    result.warnings.foreach(printMessage)
+    val dirErr = result.warnings.filter(w => w.startsWith("Error: Directory"))
```

Comment:
Not sure what this check does.

Reply:
Schema version errors happen due to either
1 - Input directory not containing schema version 1-0-0 or
2 - Input directory containing gap(s) between schema versions

per [issue strategy](https://github.com/snowplow/iglu/issues/221#issuecomment-353962880) which indicates to refuse to do anything if either 1 or 2 occur.

Their error messages start with "Error: Directory..." and I want to check if that happened, before outputting anything else. I didn't want to insert any exit mechanism to [transformSelfDescribing](https://github.com/snowplow/iglu/blob/release/r8/0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/GenerateCommand.scala#L91) hence the current implementation. It doesn't sound much clever to continue program after [this line](https://github.com/snowplow/iglu/blob/release/r8/0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/GenerateCommand.scala#L96) but `outputResult` is only place to look at for outputting any end result. Sure it would take shorter to exit if I had inserted. Also, current check is a loose connection with #221 and error outputting which could be a hidden tiny bug in future error-related issues.  Which way should we follow @chuwy ?